web/docs/understand/formats/zeek-tsv.md

```diff
@@ -0,0 +1,69 @@
+# zeek-tsv
+
+Reads and writes Zeek tab-separated values.
+
+## Synopsis
+
+```
+zeek-tsv
```

Comment:
As of now, the proposed `zeek-tsv` implementation has an optional `(<set separator>, <empty field>, <unset field>)` parameter. Should this go into the documentation or should the parameter be removed?

Reply:
Yes, this should go in there. In your PR I argued that having them as positional arguments is confusing. It also doesn't make sense to either require all or none, but rather decide on some of the individually. So my suggestion would be to use `--option=value` flags instead.

If there's no objection, I'm happy to document these options first so that you can use them as blueprint for the implementation, although it's pretty clear in this case.

Reply:
No objection from me then.

Reply:
Pushed with the latest commit.